awsapprunner/1.0.0/README.md

```diff
@@ -0,0 +1,82 @@
+# AWS App Runner Integration
+
+Keptn can integrate with AWS App Runner in 3 ways:
+
+1. Fire-and-forget triggering of App Runner from Keptn
+2. Fire-and-wait triggering of App Runner from Keptn
+3. Trigger Keptn from App Runner
+
+
+# Fire and Forget Trigger App Runner
+
+![](https://raw.githubusercontent.com/keptn-sandbox/artifacthub/main/aws-app-runner/assets/fire-and-forget-sequence.svg)
+
+This is the most basic integration. Trigger AppRunner from a Keptn task event.
+
+The Keptn [webhook service](https://github.com/keptn/keptn/tree/master/webhook-service) is used to listen for the relevant task event and trigger the App Runner endpoint. Configure the [webhook service](https://github.com/keptn/keptn/tree/master/webhook-service) to send the finished event automatically.
+
+The webhook service will:
+
+1. Listen for your event (usually a `task.triggered` event)
+1. Send a `task.started` event back to Keptn automatically
+1. Trigger AppRunner
+1. Immediately send a `task.finished` event with a `pass` result
+
+This is easy and very quick to get started but it has two behaviours that should be known.
+
+1. AppRunner is always assumed to finish immediately
+1. AppRunner is always assumed to finish successfully
+
+![](https://raw.githubusercontent.com/keptn-sandbox/artifacthub/main/aws-app-runner/assets/fire-and-forget-webhook.png)
+
+# Fire and Wait Trigger App Runner
+
+![](https://raw.githubusercontent.com/keptn-sandbox/artifacthub/main/aws-app-runner/assets/fire-and-wait-sequence.svg)
```

Comment:
This should be `https://raw.githubusercontent.com/keptn-sandbox/artifacthub/main/awsapprunner/1.0.0/assets/fire-and-wait-sequence.svg`